.zenodo.json

```diff
@@ -0,0 +1,95 @@
+{
+  "title": "MNE-BIDS: An MNE project for organizing and formatting MEG and EEG data according to the BIDS specification.",
```

Comment:
should we include iEEG here

Reply:
I changed this to

"MNE-BIDS: Organizing MEG, EEG, and iEEG data according to the BIDS specification and facilitating their analysis with MNE-Python." ... I also use this as the description to be displayed on PyPi 

I think that's a good move, because the package has moved beyond merely "organizing/formatting" and it's now also being used for facilitating analysis ... see e.g., within the mne-study-template.